.github/workflows/build-prod-container.yml

```diff
@@ -192,7 +194,7 @@ jobs:
         with:
           context: .
           file: ./images/Dockerfile
-          push: ${{ github.repository == 'PelicanPlatform/pelican' && github.event_name != 'pull_request' }}
+          push: ${{ github.repository == 'PelicanPlatform/pelican' && (inputs.release_workflow || false) }}
```

Comment:
What's with the `|| false` here? Kinda seems superfluous?

Reply:
it does look superfluous but otherwise the yaml check will fail when the workflow was not triggered by the workflow call (in which case `release_workflow` is undefined) and gives the error:

```
Error: Input does not meet YAML 1.2 "Core Schema" specification: push
Support boolean input list: `true | True | TRUE | false | False | FALSE`
```